### ISSUE TITLE
Feature request: duplicate button

### ISSUE THREAD

**Issue by allcaps** [plone/plone.app.mosaic#148]
Something like this:
<img width="871" alt="screen shot 2015-10-14 at 14 35 23" src="https://cloud.githubusercontent.com/assets/1969342/10482456/f021e4b6-7280-11e5-9d37-b5d07b953abd.png">
To duplicate selected content.

**Comment by datakurre** [comment]
That toolbar is getting crowded and adding it next to remove sounds a bit scary, but otherwise should be possible.

**Comment by vangheem** [comment]
I don't think this is necessary. Copy and paste for text tiles should be good enough.

**Comment by datakurre** [comment]
I agree. An easy way to add new pre-configured tiles (text tiles and other) would be fix this.
